sirepo_bluesky/shadow_detector.py

```diff
@@ -200,6 +202,8 @@ def update_components(_data):
             self._resource_document["resource_kwargs"]["histogram_bins"] = nbins
             update_components(ret)
 
+        print(self.view_sirepo_components)
+
```

Comment:
I think those two lines can be removed. First, that prints the method without calling it, and second, we don't want to pollute the output in production (we can always call that method to get the info).
```suggestion
```